### PR TITLE
LIIKUNTA-27 | chore: update unified-search & venues subgraph, supergraph and types

### PR DESCRIPTION
**Pull request by karisal-anders**
## Description

Depends on:
 - https://github.com/City-of-Helsinki/unified-search/pull/108 which depends on https://github.com/City-of-Helsinki/unified-search/pull/107
 - https://github.com/City-of-Helsinki/unified-search/pull/109

### chore: update unified-search & venues subgraph, supergraph and types

update unified-search & venues subgraphs, compose supergraph and update
generated types after the following changes:
 - https://github.com/City-of-Helsinki/unified-search/pull/108
 - https://github.com/City-of-Helsinki/unified-search/pull/109

refs LIIKUNTA-27 (unified search PR #109)
refs LIIKUNTA-344 (unified search PR #109)
refs US-113 (unified search PR #108)

### Related

**[LIIKUNTA-27](https://helsinkisolutionoffice.atlassian.net/browse/LIIKUNTA-27)**
**[LIIKUNTA-344](https://helsinkisolutionoffice.atlassian.net/browse/LIIKUNTA-344)**
**[US-113](https://helsinkisolutionoffice.atlassian.net/browse/US-113)**

## Additional notes

[LIIKUNTA-27]: https://helsinkisolutionoffice.atlassian.net/browse/LIIKUNTA-27?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[US-113]: https://helsinkisolutionoffice.atlassian.net/browse/US-113?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[LIIKUNTA-344]: https://helsinkisolutionoffice.atlassian.net/browse/LIIKUNTA-344?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**Federation Router branch is deployed to platta: https://events-graphql-federation-events-pr510.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Federation Router branch is deployed to platta: https://events-graphql-federation-hobbies-pr510.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Federation Router branch is deployed to platta: https://events-graphql-federation-sports-pr510.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr510.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Events-Helsinki branch is deployed to platta: https://tapahtumat-pr510.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr510.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://liikunta-pr510.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tapahtumat-pr510.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr510.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**EVENTS GRAPHQL PROXY is deployed to platta: https://events-graphql-proxy-pr510.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**VENUE GRAPHQL PROXY is deployed to platta: https://venue-graphql-proxy-pr510.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Federation Router branch is deployed to platta: https://events-graphql-federation-events-pr510.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Federation Router branch is deployed to platta: https://events-graphql-federation-sports-pr510.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Federation Router branch is deployed to platta: https://events-graphql-federation-hobbies-pr510.dev.hel.ninja** :rocket::rocket::rocket:

File: packages/components/src/types/generated/graphql.tsx [hunk 1]
```diff
@@ -11137,13 +11135,23 @@ export type UnifiedSearchVenue = {
...
   resources: Array<Resource>;
   serviceOwner?: Maybe<ServiceOwner>;
   targetGroups?: Maybe<Array<Maybe<TargetGroup>>>;
 };
 
+/**
+ * A place that forms a unit and can be used for some specific purpose -
+ * respa unit or resource, service map unit, beta.kultus venue, linked
+ * events place, Kukkuu venue
+ */
```
Comment (karisal-anders) [inline packages/components/src/types/generated/graphql.tsx:11150]: This comment is automatically copied by `yarn generate:graphql` from the comment of UnifiedSearchVenue type, see https://github.com/City-of-Helsinki/unified-search/blob/578d112907e4da1aaed0d6543dd031759a24b875/graphql/src/schemas/location.ts#L3-L8

**Comment by terovirtanen** [comment]
**Events-Helsinki branch is deployed to platta: https://tapahtumat-pr510.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr510.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tapahtumat-pr510.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr510.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr510.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**EVENTS GRAPHQL PROXY is deployed to platta: https://events-graphql-proxy-pr510.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**VENUE GRAPHQL PROXY is deployed to platta: https://venue-graphql-proxy-pr510.dev.hel.ninja** :rocket::rocket::rocket:

**Review by nikomakela — APPROVED** [review]

**Pull request merged**